tests/e2e/capsule-feature/02-install-operator.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: kuttl.dev/v1beta1
 kind: TestStep
 commands:
   - command: |
-      helm install --set image.repository=${E2E_IMAGE_REPOSITORY}
+      helm install --set image.repository=${CONTAINER_REGISTRY_URL}/${CONTAINER_REGISTRY_SPACE}/${E2E_IMAGE_REPOSITORY}
```

Comment:
CONTAINER_REGISTRY_URL, CONTAINER_REGISTRY_SPACE - how do you pass this to CI?

Reply:
We don't need to update the `make e2e` command?

Reply:
already updated, wait for check it on github actions

Reply:
fixed